newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js

```diff
@@ -46,6 +48,7 @@ type Props = {|
   onInstall: () => Promise<void>,
   onEdit?: () => void,
   alreadyInstalled: boolean,
```

Comment:
If you pass the project, the dialog can now infer if the extension is alreadyInstalled. So we should probably remove this prop to avoid any risk of inconsistency?